flit_core/flit_core/_vendor/tomli/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""A lil' TOML parser."""
+
+__all__ = ("loads", "load", "TOMLDecodeError")
+__version__ = "1.2.2"  # DO NOT EDIT THIS LINE MANUALLY. LET bump2version UTILITY DO IT
```

Comment:
Version 1.2.3 has been released now. It fixes one bug. We may want to update Tomli to that.